doc/xml/reference.xml

```diff
@@ -155,7 +155,15 @@
                     <config-key id="compress-level" name="Compress Level">
                         <summary>File compression level.</summary>
 
-                        <text>Sets the level to be used for file compression when <setting>compress-type</setting> does not equal <id>none</id> or <setting>compress=y</setting> (deprecated).</text>
+                        <text>Sets the level to be used for file compression when <setting>compress-type</setting> does not equal <id>none</id> or <setting>compress=y</setting> (deprecated).
+
+                        The following are the defaults levels based on the type when compress-level is not specified:
+                        <ul>
```

Comment:
I think this should be something like:
```
based on <setting>compress-type</setting> when <setting>compress-level</setting> is not specified
```

Reply:
Fixed 18b612b10f4d65affa901171bfa468546955f46e